stdlib/grpc/src/main/ballerina/src/grpc/service_endpoint.bal

```diff
@@ -27,40 +27,63 @@ public type Listener object {
     private int port = 0;
     private ListenerConfiguration config = {};
 
-    # Starts the registered service.
-    #
-    # + return - Returns an error if encounters an error while starting the server, returns nil otherwise.
+# Starts the registered service.
+# ```ballerina
+# error? result = listenerEp.__start();
+# ```
+#
+# + return - An `error` if an error occurs while starting the server or else `()`
     public function __start() returns error? {
         return externStart(self);
     }
 
+# Stops the service listener gracefully. Already-accepted requests will be served before the connection closure.
+# ```ballerina
+# error? result = listenerEp.__gracefulStop();
+# ```
+#
+# + return - An `error` if an error occurred during the listener stopping process or else `()`
     public function __gracefulStop() returns error? {
         return ();
     }
 
-    # Stops the registered service.
-    #
-    # + return - Returns an error if encounters an error while stopping the server, returns nil otherwise.
+# Stops the registered service.
+# ```ballerina
+# error? result = listenerEp.__immediateStop();
+# ```
+#
+# + return - An `error` if an error occurs while stopping the server or else `()`
     public function __immediateStop() returns error? {
         return externStop(self);
     }
 
-    # Gets called every time a service attaches itself to this endpoint - also happens at module init time.
-    #
-    # + s - The type of the service to be registered.
-    # + name - Name of the service.
-    # + return - Returns an error if encounters an error while attaching the service, returns nil otherwise.
+# Gets called every time a service attaches itself to this endpoint - also happens at module init time.
+# ```ballerina
+# error? result = listenerEp.__attach(helloService);
+# ```
+#
+# + s - The type of the service to be registered
+# + name - Name of the service
+# + return - An `error` if encounters an error while attaching the service or else `()`
```

Comment:
```suggestion
# + return - An `error` if encounters an error while attaching the service or else `()`
```
Need to specify the error.

Reply:
We also need to change the error type in the function signature. It will be an API change.